docs/src/network.md

```diff
@@ -421,6 +421,17 @@ await page.RouteAsync("**/api/fetch_data", async route => {
 await page.GotoAsync("https://example.com");
 ```
 
+#### Precedence
+
+When a request matches multiple routes, the last matched route is used to handle the request. As an example, the routes below will return data for `useful-resource`, and block all other requests to the API.
+
+```js
```

Comment:
Sorry for the slow review. Now we need the snippets for the other languages as well, as below.

Reply:
Thanks for the review. I'm not familiar with the other languages, so I'll close the issue for now.